applications/FluidDynamicsApplication/python_scripts/GenerateCN.py

```diff
@@ -0,0 +1,109 @@
+from math import *
```

Comment:
This file does not contain any Kratos related stuff.

 I think a discussion point for this PR is where to place these utilities.

Reply:
I would put them in `python_scripts/utilities`, esp in apps that have already many files like the FDyn App

Check e.g. the CoSimApp, there we have many subfolders as it would be a complete mess otherwise